blog/_posts/2018-09-19-wrap-up-events-2018.md

```diff
@@ -18,5 +18,22 @@ It's September 19th and we are less than two weeks away for RGSoC to be over —
 So let's celebrate all of this! Let's celebrate us getting together and making tech and OSS more inclusive for everyone! Let's celebrate making dreams come true! Wherever we are, let's organize an event — it can be big or small — and invite our friends. We created a list on [this repo](https://github.com/rails-girls-summer-of-code/announcements), so that you can add your even info or check for information about any events in your location.
 
 Another important thing: let's keep in touch! When you publish your posts on social media about the events or even messages that you want to share about the summer of code, please add the hashtag #RGSoC so that we can all see it.
+<br>
+<br>
+
+## Berlin Event at ThoughtWorks
+
+**Who is welcome?**: everyone is welcome! Supporters, sponsors, past and current participants, mentors, coaches! <br>
+**What**: An evening with food, soft drinks, and time to chat. <br>
+**When**: Saturday, 29th of September 2018, 6–9pm CEST. <br>
+**Where**: [Thoughtworks Berlin](https://www.thoughtworks.com/locations/berlin) (Zimmerstr. 23, near Checkpoint Charlie). The venue is wheelchair-accessible.  <br>
+**Drinks/Food**: This will be a no-alcohol event, there will be vegetarian food and several non-alcoholic drinks. Please let us know about any food allergies or dietary restrictions in advance, so we can cater to you.  <br>
+
+### RSVP
+If you'd like to attend, please [RSVP here](https://github.com/rails-girls-summer-of-code/announcements/issues/23) by **Wednesday, September 26th** with a comment (and let us know if you are bringing a +1 +2 +3...).
```

Comment:
if we want to include people who don't have a GH account please feel free to add a line about RSVPing by email! Something like, “if you don't have a GitHub account or prefer not sharing this information publicly, you can also RSVP by email to laura@travis-ci.org”